crates/bevy_render/src/camera/projection.rs

```diff
@@ -51,6 +51,17 @@ pub enum WindowOrigin {
     BottomLeft,
 }
 
+#[derive(Debug, Clone, Reflect, Serialize, Deserialize)]
+#[reflect_value(Serialize, Deserialize)]
+pub enum ScalingMode {
+    // Match the window size. 1 world unit = 1 pixel.
+    WindowSize,
+    // Keep vertical axis constant; resize horizontal with aspect ratio.
+    FixedVertical,
+    // Keep horizontal axis constant; resize vertical with aspect ratio.
+    FixedHorizontal,
```

Comment:
May as well have `ScalingMode::None` too, if we're here.

Reply:
hmm... what would that mean? That nothing should happen on window resize (let the image stretch), just letting the user set the `left`/`right`/`top`/`bottom` fields manually?

Reply:
Yeah, exactly that. No point limiting the functionality.